libraries/abstractions/common_io/include/iot_spi.h

```diff
@@ -155,183 +163,257 @@ void iot_spi_set_callback( IotSPIHandle_t const pxSPIPeripheral,
 /**
  * @brief Configures the SPI port with user configuration.
  *
- * The application should call this function to configure the SPI driver
- * with values in the structure IotSPIMasterConfig_t. The application also needs to
- * specify the request which is as per enum IotSPIIoctlRequest_t.
+ *
+ * @note eSPISetMasterConfig sets the configurations for master.
+ * This request expects the buffer with size of IotSPIMasterConfig_t.
+ *
+ * @note eSPIGetMasterConfig gets the current configuration for SPI master.
+ * This request expects the buffer with size of IotSPIMasterConfig_t.
+ *
+ * @note eSPIGetTxNoOfbytes returns the number of written bytes in last operation.
+ * This is supposed to be called in the caller task or application callback, right after last operation completes.
+ * This request expects 2 bytes buffer (uint16_t).
+ *
+ * - If the last operation only did write, this returns the actual number of written bytes which might be smaller than the requested number (partial write).
+ * - If the last operation only did read, this returns 0.
+ * - If the last operation did both write and read, this returns the number of written bytes.
+ *
+ * @note eSPIGetRxNoOfbytes returns the number of read bytes in last operation.
+ * This is supposed to be called in the caller task or application callback, right after last operation completes.
+ * This request expects 2 bytes buffer (uint16_t).
+ *
+ * - If the last operation only did read, this returns the actual number of read bytes which might be smaller than the requested number (partial read).
+ * - If the last operation only did write, this returns 0.
```

Comment:
line #181 - 184 is this duplicate of line no 177-179?

Reply:
yes, I think that is OK? One is Tx and the other is Rx.

Reply:
ok